aptos-move/framework/aptos-framework/sources/transaction_validation.move

```diff
@@ -263,10 +282,11 @@ module aptos_framework::transaction_validation {
         storage_fee_refunded: u64,
         txn_gas_price: u64,
         txn_max_gas_units: u64,
-        gas_units_remaining: u64
+        gas_units_remaining: u64,
+        is_simulation: bool,
```

Comment:
Not in this PR but it might be good to start using a struct for this, it's a lot of args.

Reply:
It's a good point!